<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1750</width>
    <height>973</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="main_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1151</width>
      <height>921</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QVTKWidget" name="vtk_main" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1151</width>
       <height>921</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_iso">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>690</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Level Sets</string>
    </property>
    <widget class="QCheckBox" name="show0_checkbox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>88</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Show</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="value0_spinbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>67</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>34</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>value 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>63</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>value 1</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="value1_spinbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>67</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="show1_checkbox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>88</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Show</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_x">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>0</y>
      <width>301</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>X slice</string>
    </property>
    <widget class="QVTKWidget" name="vtk_x" native="true">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>19</y>
       <width>301</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_y">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>230</y>
      <width>301</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Y slice</string>
    </property>
    <widget class="QVTKWidget" name="vtk_y" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>301</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_z">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>460</y>
      <width>301</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Z slice</string>
    </property>
    <widget class="QVTKWidget" name="vtk_z" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>301</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_range">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>800</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Range</string>
    </property>
    <widget class="QDoubleSpinBox" name="min_spinbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>67</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="max_spinbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>67</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>40</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>min</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>63</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>max</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_measure">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>0</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Measure</string>
    </property>
    <widget class="QTextEdit" name="label_edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Label</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="ID_spinbox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Kernel ID</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_log">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>300</y>
      <width>281</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Log</string>
    </property>
    <widget class="QTextEdit" name="log_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>261</width>
       <height>151</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>110</y>
      <width>281</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Info</string>
    </property>
    <widget class="QPlainTextEdit" name="bounds_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bounds</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="measure_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Measure</string>
     </property>
    </widget>
    <widget class="QPushButton" name="compute_push">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Compute</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_view">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>530</y>
      <width>281</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QPushButton" name="front_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Front</string>
     </property>
    </widget>
    <widget class="QPushButton" name="top_button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Top</string>
     </property>
    </widget>
    <widget class="QPushButton" name="left_button">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Left</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bottom_button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Bottom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="right_button">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1750</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
